x/auth/ante.go

```diff
@@ -236,6 +244,15 @@ func consumeSignatureVerificationGas(meter sdk.GasMeter, pubkey crypto.PubKey) {
 	}
 }
 
+func calculateFeesByGas(fees sdk.Coins, gas int64) sdk.Coins {
```

Comment:
Hmm OK I don't think this really works with multiple-denomination fees, we don't necessarily want the same amounts for each.

We don't need multiple-denom fees at launch anyways so maybe better to assert single-denom for now.

Reply:
ditto